setup.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+
+import setuptools
+
+ROOT = os.path.dirname(__file__)
+
+if sys.version_info < (3, 6, 0):
+    sys.exit("Python 3.6.0 is the minimum required version")
```

Comment:
why do we need it?

Reply:
When test infra will use other version we can move that too

Reply:
`setuptools>=51.0.0` needs python 3.6 as minimal version, so it is needed for build (https://pypi.org/project/setuptools/)

Reply:
so please update the message
thanks!